ghost/core/test/utils/e2e-utils.js

```diff
@@ -167,6 +172,10 @@ const freshModeGhostStart = async (options) => {
 
     await settingsService.init();
 
+    // We need to clear the adapter cache because the config changed
+    // And the LocalStorageAdapter is cached, which means that it will still try to store files to the old content path if we don't recreate all adapters
+    adapterManager.clearCache();
```

Comment:
Few things here:
1. Probably best to place the clear cache call above the "settingsService.init" just in case when settings service starts using adapters - cache adapter for example. Wdyt?

2. The comment provides too many details about the "why" we are doing the "clearCache" 🙃  When looking at it out of context I would need to figure out how LocalStorageAdapter is related here specifically etc.  I think just saying that the "adapter cache has to be cleared to avoid reusing cached adapter instances between restarts" would be enough 👍